src/dvc_data/hashfile/transfer.py

```diff
@@ -147,7 +151,6 @@ def _add(
     hash_infos: Iterable["HashInfo"],
     **kwargs,
 ) -> Set["HashInfo"]:
-
```

Comment:
Looks like this change is still here, can't reproduce locally, so maybe a leftover from your editor or something?

Reply:
It's definately not my editor. For whatever reason, when I run black that extra line gets removed. In the repo directory I ran:

```bash
black ~/code/dvc-data/src/dvc_data/hashfile/transfer.py
```

Not sure if it's not respecting some repo rule? Either way it's easy enough to  fixup manually.